### PR TITLE
[athena_1]: complete the task_1

### PR DESCRIPTION
**Pull request by martinx**
### What?

brief: task_1 完成
author:  killing11l, badmicroctrl, martinx

### What changes are included in this PR?

* 完成stl_list代码学习
* 整理并编写学习笔记
* 编写测试代码

### Are these changes tested?

已测试

### CODE REVIEW

File: stl/athena/killing11l/doc/task_1/README.md [hunk 1]
```diff
@@ -1 +1,63 @@
-## Task1 的学习笔记
\ No newline at end of file
+## Task1 的学习笔记
+# std::List默认构造器std::allocator<T>
```
Comment (Light-City) [inline stl/athena/killing11l/doc/task_1/README.md:2]: markdown grammer sequence.

#. -> first-level title
## ->  second-level
###....

File: stl/athena/killing11l/src/task_1/main.cpp
Comment (Light-City) [inline stl/athena/killing11l/src/task_1/main.cpp:1]: good example.

File: stl/athena/martinx/src/task_1/stock.cpp [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+// 使用股票代码初始化
+Stock::Stock(const std::string& code) { this->code = code; };
+
+// 获取股票最后一个交易日的分钟K线数据
+std::string Stock::GetData() {
+  std::string fields = "time,price,volume,avg_price,amount,highest,lowest";
+
+  // 创建 HTTP 客户端
+  httplib::Client cli("yunhq.sse.com.cn", 32041);
```
Comment (Light-City) [inline stl/athena/martinx/src/task_1/stock.cpp:16]: use macro may be a good idea.

Comment (Light-City) [reply stl/athena/martinx/src/task_1/stock.cpp:16]: no magic

File: stl/athena/badmicroctrl/doc/task_1/README.md
Comment (Light-City) [inline stl/athena/badmicroctrl/doc/task_1/README.md:1]: why your list content is empty?

Comment (badmicroctrl) [reply stl/athena/badmicroctrl/doc/task_1/README.md:1]: you mean the table? misunderstanding leads to this, already filled

File: stl/athena/killing11l/doc/task_1/README.md (hunk 1 above)
Comment (Killing11L) [reply stl/athena/killing11l/doc/task_1/README.md]: Ok, the modification has been submitted to the team for review.

**Comment by Light-City** [comment]
LGTM

**Pull request merged**